### ISSUE TITLE
Required Fields

### ISSUE THREAD

**Issue by spsdevice** [snipe/snipe-it#4528]
#### Expected Behavior (or desired behavior if a feature request)

I expected to just put in a new asset number, who it is checked out to and the location. However, I cannot proceed forward when creating a new asset, unless I select an asset to be checked out to. Is there a way around this? Can I disable the required field?

-----

#### Actual Behavior

When I create an asset, it requires me to input/select another asset to check out to.
-----

#### Please confirm you have done the following before posting your bug report:

- [ ] I have enabled debug mode - Yes
- [ ] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues) - Yes

-----
#### Provide answers to these questions:

- Is this a fresh install or an upgrade?
An Upgrade
- Version of Snipe-IT you're running
4.13
- Version of PHP you're running
PHP 7.0
- Version of MySQL/MariaDB you're running

- What OS and web server you're running Snipe-IT on
Ubuntu 16.04
- What method you used to install Snipe-IT (install.sh, manual installation, docker, etc)
install.sh
- WITH DEBUG TURNED ON, if you're getting an error in your browser, include that error
No Error
- What specific Snipe-IT page you're on, and what specific element you're interacting with to trigger the error
No error, just a request
- If a stacktrace is provided in the error, include that too.
None
- Any errors that appear in your browser's error console.
None
- Confirm whether the error is reproduceable on the demo: https://snipeitapp.com/demo.
Yes
- Include any additional information you can find in `app/storage/logs` and your webserver's logs.
None
- Include what you've done so far in the installation, and if you got any error messages along the way.
None
- Indicate whether or not you've manually edited any data directly in the database
No

Please do not post an issue without answering the related questions above. If you have opened a different issue and already answered these questions, answer them again, once for every ticket. It will be next to impossible for us to help you.

https://snipe-it.readme.io/docs/getting-help

**Comment by snipe** [comment]
> unless I select an asset to be checked out to.

When creating a new asset, it's not required to be checked out. Can you submit a screenshot to explain what you mean.

**Comment by spsdevice** [comment]
Nevermind, it looks like this got resolved after I upgraded to v 4.1.6 
But in version 4.1.3, The field that has "Check out to: Asset", an orange bar is next to it, leaving me to believe that it is required. So it left some confusion with my users when they are creating a new asset. 

![image](https://user-images.githubusercontent.com/32550757/33290141-a737a00c-d376-11e7-9225-22f02894fc0b.png)

And there was an issue, when you selected the model number and it comes out with a "The results could not be loaded"

![image](https://user-images.githubusercontent.com/32550757/33290216-d9735ac0-d376-11e7-9b5c-92a2ba0db62c.png)

But it looks like all this went away after I upgraded. This new version is way nicer. Thanks for your prompt update!
